README.md

```diff
@@ -26,24 +26,36 @@ from armorblox import client
 c = client.Client(api_key='your-api-key-here', instance_name='yourtenantname')
 
 # Fetch a list of threats
-threat_incidents = c.threats.list()
+threat_incidents, next_page_token, total_incident_count = c.threats.list()
 
 # Fetch a specific threat
 incident = c.threats.get(44006)
 
 
 # Fetch a list of abuse incidents
-abuse_incidents = c.abuse_incidents.list()
+abuse_incidents, next_page_token, total_incident_count = c.abuse_incidents.list()
 
 # Fetch a specific abuse incident
 abuse_incident = c.abuse_incidents.get(44200)
 
 
 # Fetch a list of DLP incidents
-dlp_incidents = c.dlp_incidents.list()
+dlp_incidents, next_page_token, total_incident_count = c.dlp_incidents.list()
 
 # Fetch a specific DLP incident
 dlp_incident = c.dlp_incidents.get(44010)
+
+
+# Example to fetch all threats using next_page_token
+c = client.Client(api_key='your-api-key-here', instance_name='yourtenantname')
+next_page_token = None
```

Comment:
You can set this to `True` and then skip lines 53 & 54 completely.

Reply:
Actually, that still might have an issue. You can keep this as it is and just do `while True` in the loop and add an if check for the token being None and `break` if that's the case.

Reply:
Right, makes sense. I will update it.